Swift-Base/Constants.swift

```diff
@@ -12,6 +12,10 @@ import UIKit
 let kAPIKeyCrashlitycs  = ""
 let kAPIKeyGoogle       = ""
 
+/*--------------User Defaults keys-------------*/
+let kUserDefaultsDeviceTokenData = "kUserDefaultsDeviceTokenData"
+let kUserDefaultsDeviceTokenString = "kUserDefaultsDeviceTokenString"
+
 /*----------Notifications---------*/
 let kMockNotification:String = "mock_notification";
```

Comment:
We must use some prefix for notifications for autocomplete.
Maybe not exactly "k". I think we must discuss this at Wednesday